docs/pages/deployment/cli/commands/nuts_network.rst

```diff
@@ -0,0 +1,34 @@
+.. _nuts_network:
```

Comment:
```
nuts network -h
```
returns
```
network commands

Usage:
  nuts network [command]

Available Commands:
  get         Gets a transaction from the network
  list        Lists the transactions on the network
  payload     Retrieves the payload of a transaction from the network
  peers       Get diagnostic information of the node's peers
  reprocess   Reprocess all transactions with the give contentType (ex: application/did+json)

Flags:
      --address string     Address of the remote node. Must contain at least host and port, URL scheme may be omitted. In that case it 'http://' is prepended. (default "localhost:1323")
  -h, --help               help for network
      --timeout duration   Client time-out when performing remote operations, such as '500ms' or '10s'. Refer to Golang's 'time.Duration' syntax for a more elaborate description of the syntax. (default 10s)
      --verbosity string   Log level (trace, debug, info, warn, error) (default "info")
```

The `Available Commands` section from the help are in the docs replaced by the `SEE ALSO` section and moved to the end. To me it was not clear that this section contains the available commands and looks more like a suggested reading section that I can ignore. Can this be changed to look a bit more like the help?